leaf-ui/js/backendComm.js

```diff
@@ -189,7 +174,7 @@ function getFileResults(isGetNextSteps){
                     analysisResult.allSolution = results.allSolution;
                     analysisRequest.previousAnalysis = analysisResult;
                     console.log("previousAnalysis");
-                    console.log(analysisRequest.previousAnalysis);
+					console.log(analysisRequest.previousAnalysis);
```

Comment:
stop being dramatic github

Reply:
Yeah lmfao I went back when I saw this to be like what is going on and the extra spaces arent even showing up in my IDE lol